llmv_models.py

```diff
@@ -16,6 +17,22 @@
 
 log = getLogger(__name__)
 
+class LLMVChatPair(db.Model):
+    __tablename__ = 'llmv_chat_pair'
+    __table_args__ = {'extend_existing': True}
+    id = db.Column(db.Integer, primary_key=True)
+    generation_id = db.Column(db.Integer, db.ForeignKey('llmv_generation.id', ondelete='CASCADE'))
+    prompt = db.Column(db.Text)
+    generation = db.Column(db.Text)
```

Comment:
Consider making these nullable in case the model fails for some reason and a generation is not returned